pom.xml

```diff
@@ -14,4 +14,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.8.1</version>
```

Comment:
I saw your comment, but it looks like this dependency is redundant.
Did your code after the initial commit work? If not can you clarify what exactly did not work, why did your project fail to compile?